package.json

```diff
@@ -77,29 +77,31 @@
     "@testing-library/jest-dom": "^5.17.0",
     "@testing-library/user-event": "^14.4.3",
     "@testing-library/vue": "^7.0.0",
-    "@types/lodash": "^4.14.195",
+    "@types/lodash": "^4.14.196",
     "@types/mdx": "^2.0.4",
-    "@typescript-eslint/eslint-plugin": "^5.59.11",
-    "@typescript-eslint/parser": "^5.62.0",
+    "@typescript-eslint/eslint-plugin": "^6.3.0",
+    "@typescript-eslint/parser": "^6.3.0",
     "@vitejs/plugin-vue": "^4.2.3",
-    "@vitest/ui": "^0.33.0",
+    "@vitest/ui": "^0.34.1",
     "@vue/eslint-config-prettier": "7.1.0",
     "@vue/eslint-config-typescript": "^11.0.3",
     "autoprefixer": "^10.4.14",
     "chromatic": "^6.20.0",
-    "eslint": "^8.45.0",
+    "eslint": "^8.46.0",
+    "eslint-config-prettier": "^9.0.0",
```

Comment:
a new dependency?

Reply:
I installed it to resolve a peer conflict and forgot to remove it again, thanks [a992c1b](https://github.com/archilogic-com/honeycomb/pull/67/commits/a992c1bc3040977bc41b1d08badf3aca3b737899)

```sh
@archilogic/honeycomb@2.4.2 
├─┬ @archilogic/eslint-config@2.0.0
│ └── eslint-config-prettier@9.0.0
└─┬ @vue/eslint-config-prettier@7.1.0
  └── eslint-config-prettier@8.10.0
```